server/xpub-controller/manuscript.js

```diff
@@ -56,6 +55,8 @@ class Manuscript {
         progress,
         manuscriptId,
       )
+      const manuscript = await ManuscriptModel.find(manuscriptId, this.userId)
+      await ManuscriptHelper.clearPendingFile(manuscript)
```

Comment:
It may be worth doing this in a `finally` statement. If for some reason an error gets thrown when we call `endFileProgress`, the file would be left in a pending state.

Reply:
Happy for this to happen in a separate PR btw.

Reply:
As we're going to be re-thinking this controller code lets leave this till then. This case fixes the bug that is currently in production and we can consider where else we might need the clean up task ran in the controller.